File: 3ra parte/kotlin/kotlin-end/app/src/main/java/com/unaj/tallerunajweatherapp/model/MainWeatherModel.kt
package com.unaj.tallerunajweatherapp.view.model

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import com.unaj.tallerunajweatherapp.model.TempModel
import com.unaj.tallerunajweatherapp.model.WeatherModel
import kotlinx.android.parcel.Parcelize

@Parcelize
data class MainWeatherModel (
    @SerializedName("name")
    val cityName: String,
    @SerializedName("weather")
    val description: List<WeatherModel>,
    @SerializedName("main")
    var temperature: TempModel
): Parcelable

File: 3ra parte/swift/swift-start/TallerUNAJWeatherApp/Model/MainWeatherModel.swift
//
//  WeatherModel.swift
//  TallerUNAJWeatherApp
//
//  Created by itsupport on 16/08/2020.
//  Copyright © 2020 UNAJ. All rights reserved.
//

import Foundation

struct MainWeatherModel {
    let cityName: String
    let weather: [WeatherModel]
    var temperature: TempModel
    
    init(cityName: String, weather: [WeatherModel], temperature: TempModel) {
        self.cityName = cityName
        self.weather = weather
        self.temperature = temperature
    }
}

